src/Microsoft.ML.Core/Microsoft.ML.Core.csproj

```diff
@@ -8,10 +8,6 @@
     <RootNamespace>Microsoft.ML</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="Properties\" />
```

Comment:
>Properties [](start = 21, length = 10)

I believe you still need all InternalsVisibleTo attributes from Properties/AssemblyInfo.cs

Reply:
Are they not included automatically? All tests passed.

Reply:
Why then you have that ItemGroup with folder in Microsoft.ML.CpuMath?

---
In reply to: [236471881](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236471881) [](ancestors = 236471881)

Reply:
Removed.

---
In reply to: [236474207](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236474207) [](ancestors = 236474207,236471881)